benches/bench.rs

```diff
@@ -25,7 +27,41 @@ mod poly_benches {
                 };
                 b.iter_with_setup(rand_factors, |(lhs, rhs)| &lhs * &rhs)
             },
-            &[5, 10, 20, 40],
+            &TEST_DEGREES,
+        );
+    }
+
+    // Benchmarks subtraction of two polynomials
```

Comment:
Use `///` here maybe? Just a suggestions.

Reply:
Sure, I can change that, but the pre-existing benchmarks used `//`. If that wasn't intentional, I'll update all of them.